res/controllers/Numark-Party-Mix.midi.xml

```diff
@@ -0,0 +1,629 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxMIDIPreset schemaVersion="1" mixxxVersion="2.3.1">  <!-- Schema version number to help compatibility, should the MIDI format change -->
+  <info>
+    <name>Numark Party Mix</name>
+    <author>Ryli Dunlap, fiveangle, h67ma, bad1dea5, photoenix, olaf</author>
```

Comment:
I left some names in the list because they motivated me to start. But I basically started from copy pasting from this controller: https://github.com/mixxxdj/mixxx/blob/main/res/controllers/Numark%20Mixtrack%20Pro%20FX.midi.xml

Reply:
So I change the authorship accordingly.

Reply:
Ok, as long as you base your work of mappings already included in mixxx, its fine.